hadoop-hdfs-project/hadoop-hdfs/pom.xml

```diff
@@ -35,6 +35,17 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>apache-log4j-extras</artifactId>
+      <version>${log4j-extras.version}</version>
```

Comment:
declare this in hadoop-project pom with version and exclusions, then declare in this file without those. this is to ensure consistent importing across different modules